RELEASE_NOTES.md

```diff
@@ -9,41 +9,69 @@
+  | Bulgarian  | **bg-BG** |
+  | Estonian   | **et-EE** |
+  | Latvian    | **lv-LV** |
+  | Lithuanian | **lt-lT** |
 
 ## Improved
-- For UPI Intent an error message will be shown when "Continue" button is pressed without selecting
-  any UPI option.
-- For drop-in, improved accessibility of back/close button in the navigation bar.
+- For UPI Intent, if the shopper selects the **Continue** button without selecting an UPI option, an error message now shows.
+- For Drop-in, in the navigation bar, the accessibility of the **Back/Close** button is improved.
 
 ## Changed
-- For drop-in, headers of preselected stored payment screen and payment methods list screen are
-  updated.
+- For Drop-in, headers of preselected stored payment screen and payment methods list screen are updated.
+- When you set `analyticsConfiguration = AnalyticsConfiguration(AnalyticsLevel.NONE)`, only [Drop-in/Components analytics](https://docs.adyen.com/online-payments/analytics-and-data-tracking#data-we-are-collecting) are not sent to Adyen.
 - Dependency versions:
   | Name                                                                                                   | Version                       |
   |--------------------------------------------------------------------------------------------------------|-------------------------------|
-  | [Adyen 3DS2](https://github.com/Adyen/adyen-3ds2-android/releases/tag/2.2.20)                          | **2.2.20**                    |
+  | [Adyen 3DS2](https://github.com/Adyen/adyen-3ds2-android/releases/tag/2.2.21)                          | **2.2.21**                    |
+  | [Cash App Pay](https://github.com/cashapp/cash-app-pay-android-sdk/releases/tag/v2.5.0)                | **2.5.0**                     |
+  | [Android Gradle Plugin](https://developer.android.com/build/releases/past-releases/agp-8-5-0-release-notes#android-gradle-plugin-8.5.1)                          | **8.5.1**                    |
   | [AndroidX Fragment](https://developer.android.com/jetpack/androidx/releases/fragment#1.8.3)            | **1.8.3**                     |
   | [AndroidX Activity](https://developer.android.com/jetpack/androidx/releases/activity#1.9.2)            | **1.9.2**                     |
   | [AndroidX Compose Activity](https://developer.android.com/jetpack/androidx/releases/activity#1.9.2)    | **1.9.2**                     |
+  | [AndroidX Compose BOM](https://developer.android.com/develop/ui/compose/bom/bom-mapping)               | **2024.06.00**                |
+  | [AndroidX Lifecycle](https://developer.android.com/jetpack/androidx/releases/lifecycle#2.8.3)          | **2.8.3**                     |
+  | [AndroidX Lifecycle ViewModel Compose](https://developer.android.com/jetpack/androidx/releases/lifecycle#2.8.3)  | **2.8.3**                     |
+  | [AndroidX AppCompat](https://developer.android.com/jetpack/androidx/releases/appcompat#1.7.0)          | **1.7.0**                     |
+    
 
 ## Fixed
```

Comment:
One last nit (really sorry). Can we move Fixed higher for consistency with the previous release notes?